README.md

```diff
@@ -119,13 +125,17 @@ for role in "Virtual Machine Contributor" "Network Contributor" "Managed Identit
 done
 ```
 
+> Note: If you experience any issues creating the role assignments, but should have the given ownership to do so, try going through the Azure portal:
```

Comment:
I'm thinking this should maybe be removed, but it was an issue I personally ran into while going through the steps.

Reply:
Leaving in for now